api/src/authentication/models.py

```diff
@@ -15,11 +16,11 @@ def check_privilege(self, required_level: "AccessLevel") -> bool:
         return False
 
     @classmethod
-    def __get_validators__(cls):
+    def __get_validators__(cls):  # type:ignore

```

Comment:
Seems to be a mypy bug here. Did not accept any of its own suggestions, just went in circle with reccomendations. Tried with "Iterable", "Generator", ""AccessLevel"